Makefile

```diff
@@ -451,7 +451,6 @@ node_modules/angular/angular.min.js: .build/node_modules.timestamp
 	touch $@
 
 node_modules/font-awesome/fonts/fontawesome-webfont.%: .build/node_modules.timestamp
-	touch --no-create $@
```

Comment:
Should be removed, see: https://github.com/camptocamp/ngeo/pull/630#issuecomment-182773312